packaging/nix/flake.nix

```diff
@@ -2,53 +2,38 @@
   description = "umu universal game launcher";
 
   inputs = {
-    nixpkgs = {
-      type = "github";
-      owner = "NixOS";
-      repo = "nixpkgs";
-      ref = "nixpkgs-unstable";
-    };
+    # FIXME:
+    # upstream package is in nixpkgs-unstable & nixos-unstable-small
+    # it will be in nixos-unstable soon
+    nixpkgs.url = "github:nixos/nixpkgs/nixpkgs-unstable";
```

Comment:
```suggestion
    nixpkgs.url = "nixpkgs/nixos-unstable";
```

Reply:
https://github.com/Open-Wine-Components/umu-launcher/pull/345#discussion_r1921957309

Reply:
Whoops, well, the simplification is still relevant

Reply:
Doesn't seem too important, I think this mostly comes down to personal preference.

Using the shorthand `nixpkgs` syntax also relies on the flake registry; IMO that kinda impurity is bad practice, especially in a flake. Even more especially in a publicly consumed flake.

I'd much rather be explicit and say `github:nixos/nixpkgs`, rather than rely on the flake registry being setup as expected on the user's system.

Reply:
What? It doesn't rely on any sort of registry? It's hardcoded into nix

Reply:
https://nix.dev/manual/nix/2.18/command-ref/new-cli/nix3-flake.html#examples